src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -2286,6 +2286,7 @@ impl Step for RustDev {
         tarball.permit_symlinks(true);
 
         builder.ensure(crate::core::build_steps::llvm::Llvm { target });
+        builder.ensure(crate::core::build_steps::gcc::Gcc { target });
```

Comment:
Let's move the Linux x64 check out of `Gcc` and put it here. Both to make it more explicit that we're only packaging `Gcc` in a selected platform, and also to make it possible to execute the `Gcc` step manually, perhaps even on different platforms.

Reply:
Done as well.